client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -410,6 +411,21 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 		return false;
 	},
 
+	renderNoQueryArgsError() {
+		return (
+			<Main>
+				<EmptyContent
+					illustration="/calypso/images/drake/drake-whoops.svg"
+					title={ this.translate(
+						'Oops, this URL should not be accessed directly'
+					) }
+					action={ this.translate( 'Get back to Jetpack Connect screen' ) }
```

Comment:
There's also the chance to add a `line=` property with more details about why the URL fails. Do you think it's worth it ?